.circleci/config.yml

```diff
@@ -263,6 +276,7 @@ jobs:
       - run:
           name: 'Performance Tests'
           command: node build-system/pr-check/performance-tests.js
+      - save_artifacts
```

Comment:
Add this to `Experiment Tests` as well. (That job runs both integration tests and E2E tests on the experiment A/B/C binaries.)

When this works, we should see test results here: https://app.circleci.com/pipelines/github/ampproject/amphtml/6299/workflows/1e6545e8-8eef-4078-9faa-f09c97974a63/jobs/89742/tests

Reply:
Good catch!